src/mol-canvas3d/canvas3d.ts

```diff
@@ -315,6 +315,10 @@ namespace Canvas3D {
         let y = 0;
         let width = 128;
         let height = 128;
+
+        let forceNextRender = false;
+        let currentTime = 0;
```

Comment:
This was required by the ES6 target saying the variable is not declared yet?

Reply:
yes, because with es6 you have let/const